docs/posts/0000-00-00-template.md

```diff
@@ -0,0 +1,19 @@
+---
+date: '0000-00-00'
+title: 'Template title'
+description: 'Template description'
+coverUrl: '/social_preview.png'
+coverAlt: 'Cover description'
+
+aside: false
+---
+
```

Comment:
To avoid having to repeat the texts written above, these lines allow to render them again within the article itself, the idea is to follow a standard format in all our articles so these lines should not be modified, but can be modified whenever an article requires it.